apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -38,8 +41,9 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
   @early_blknum 1_000
   @late_blknum 10_000
 
-  @utxo_pos1 Utxo.position(1, 0, 0)
+  @utxo_pos1 Utxo.position(1, 3, 0)
```

Comment:
a little bizarre concept, with position numbering.
I'm confused why pos3 is before pos1